sdk/eventhub/eventhubs-checkpointstore-blob/changelog.md

```diff
@@ -1,6 +1,14 @@
+### 2019-11-05 - 1.0.0-preview.5
```

Comment:
Use `TBD` or `Coming Soon...` instead of the date

Reply:
But surely we can release in the past!?

(changed)